compiler/src/formatting/fmt.re

```diff
@@ -3985,10 +3991,36 @@ let print_program = (fmt, parsed_program) => {
     };
 
   group @@
-  fmt.print_comment_range(
-    fmt,
-    enclosing_start_location(parsed_program.prog_loc),
-    parsed_program.module_name.loc,
+  concat_map(
+    ~lead=
+      _ =>
+        fmt.print_comment_range(
+          fmt,
+          enclosing_start_location(parsed_program.prog_loc),
+          first_loc,
```

Comment:
Shouldn't this just use the argument to the function instead of trying to calculate the first_loc outside?

Reply:
Sorry, not sure what you mean; are you referring to an argument to `print_comment_range`? Not sure which one I'd want here

Reply:
No, you calculate `first_loc` outside of this function, but you don't use the argument to `~lead` like you are supposed to.